settings/dev.settings.json

```diff
@@ -3,9 +3,9 @@
   "MONGO_URL": "",
   "MAIL_URL": "",
   "reaction": {
-    "REACTION_USER": "",
-    "REACTION_AUTH": "",
-    "REACTION_EMAIL": ""
+    "REACTION_USER": "prinzdezibel@gmail.com",
```

Comment:
agreed.  👍

Sent from mobile device. Please excuse brevity and typos.

On Aug 11, 2016 2:52 AM, "Aaron Judd" <notifications@github.com> wrote:

> In settings/dev.settings.json
> <https://github.com/reactioncommerce/reaction/pull/1136#discussion_r74357465>
> :
>
> > @@ -3,9 +3,9 @@
> >    "MONGO_URL": "",
> >    "MAIL_URL": "",
> >    "reaction": {
> > -    "REACTION_USER": "",
> > -    "REACTION_AUTH": "",
> > -    "REACTION_EMAIL": ""
> > +    "REACTION_USER": "prinzdezibel@gmail.com",
>
> Oops. we don't want this in the code ;-)
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/reactioncommerce/reaction/pull/1136/files/5d08561bb80e286121894d90b4235b06eeed0c7f#r74357465>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABpybadDr0iYs5-PEm2MWmcS2Qm9Gx_Tks5qenJNgaJpZM4JGc89>
> .
>